dist/tools/pyterm/pyterm

```diff
@@ -700,6 +710,16 @@ class SerCmd(cmd.Cmd):
             elif c == self.serprompt and output == "":
                 sys.stdout.write('%c ' % self.serprompt)
                 sys.stdout.flush()
+                # we now need to read away the space we printed
```

Comment:
Maybe add a comment above on what is ignore for this whole section?

Reply:
So the `> + ' '`

Reply:
The `>` is not ignored. It is checked in the `c == self.serprompt`.

Reply:
I updated the comment in the latest force push. Is that what you had in mind?

Reply:
Better now?

Reply:
It's a bit rambling now, but at least it should be easy to understand.